MoinMoin/Page.py

```diff
@@ -1440,7 +1440,11 @@ def send_page_content(self, request, body, format='wiki', format_args='', do_cac
 
     def format(self, parser):
         """ Format and write page content without caching """
-        parser.format(self.formatter)
+        try:
+            parser.format(self.formatter)
+        except ValueError:
```

Comment:
Hmm, is it only ValueError or do we also want to catch other exceptions here? Could also use `Exception`.